src/static/js/collab_client.js

```diff
@@ -159,6 +159,9 @@ function getCollabClient(ace2editor, serverVars, initialUserInfo, options, _pad)
         rev=newRev;
         if (msg.type == "ACCEPT_COMMIT")
         {
+          // useful for knowing if a user has committed content to pad or not
+          clientVars.thisUserHasEditedThisPad = true;
```

Comment:
Please initialize this variable to `false` somewhere earlier (perhaps at first client connection?)